# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/ipc/ContentChild.cpp
# Commit: a39da23d450b
# Full Hash: a39da23d450b7e3cda053097d77ca47de40f1bb2
# Author: Nika Layzell <nika@thelayzells.com>
# Date: 2019-03-15 03:42:43
# Regressor Bug: 1532661
# File Overlap Count: 2
# Description:
#   Bug 1532661 - Part 2: Use a field list macro header for synced fields, r=farre
#   
#   Depends on D22190
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D22191
# ==============================================================================

diff -r 84d076d2163d -r a39da23d450b dom/ipc/ContentChild.cpp
--- a/dom/ipc/ContentChild.cpp	Thu Mar 14 18:51:03 2019 +0000
+++ b/dom/ipc/ContentChild.cpp	Thu Mar 14 18:51:05 2019 +0000
@@ -3794,7 +3794,7 @@
 mozilla::ipc::IPCResult ContentChild::RecvCommitBrowsingContextTransaction(
     BrowsingContext* aContext, BrowsingContext::Transaction&& aTransaction) {
   if (aContext) {
-    aTransaction.Apply(aContext);
+    aTransaction.Apply(aContext, nullptr);
   }
   return IPC_OK();
 }